CONTRIBUTING.md

```diff
@@ -185,27 +185,32 @@ For all available scripts see the package's `package.json` file.
 **Prerequisites:** [Docker][docker] 🐳
 
 You can run visual regression testing via `Makefile` in the project root.
-Using `make test-e2e` you will execute docker command that starts [Playwright][playwright] in containerized environment.
+All commands will execute docker command that starts [Playwright][playwright] in containerized environment.
+
+- Using `make test-e2e` you will run the tests.
+- Using `make test-e2e-update` you will update the snapshots.
+- Using `make test-e2e-report` you will generate and serve report of the visual regression testing.
```

Comment:
Where is the report served? Is it a URL that is automatically opened in a browser?

Reply:
Yes, the Playwright gives you the URL in the console.